### PR TITLE
Set base_price in stock.move and discounts

### PR DESCRIPTION
**Pull request by raimonesteve**
#170759

### CODE REVIEW

File: move.py [hunk 1]
```diff
@@ -4,168 +4,137 @@
...
-        with Transaction().set_context(company=self.company.id):
-            return Date.today()
+        'Taxes'), 'get_taxes')
+    amount = fields.Function(Monetary('Amount', digits='currency',
+        currency='currency'), 'on_change_with_amount')
 
     @classmethod
-    def get_origin_fields(cls, moves, names):
+    def __register__(cls, module_name):
+        # Rename gross_unit_price to base_price
```
Comment (raimonesteve) [inline move.py:59]: "gross_unit_price" was a function field. Why add rename column table?

File: move.py [hunk 2]
```diff
@@ -4,168 +4,137 @@
...
 
 class Move(metaclass=PoolMeta):
     __name__ = 'stock.move'
-    gross_unit_price = fields.Function(Monetary('Gross Price',
-        digits=price_digits, currency='currency', states=STATES),
-        'get_origin_fields')
-    amount = fields.Function(Monetary('Amount',
-        digits='currency', currency='currency'), 'get_origin_fields')
+
+    base_price = Monetary(
```
Comment (raimonesteve) [inline move.py:23]: When will the base price has a value? stock_valued module require extras-depends to sale_discount module and set when base_price when create new moves?

File: move.py (hunk 1 above)
Comment (albertca) [reply move.py:59]: You're right, should be removed this migration.

Comment (raimonesteve) [reply move.py]: Done 92bc841e0b2760579314660434fdb2801fca805a

**Review by albertca — APPROVED** [review]

**Pull request merged**